edgelet/build/linux/debian9/aarch64/test-iotedged.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
What is this file used for?

Reply:
This is so we can run unit tests on the ARM64 Rust binaries using QEMU. Intended to be a step in building the package, because we don't build tests on the release builds.